packages/cli/src/util/runner.ts

```diff
@@ -0,0 +1,38 @@
+import { getHttpOperationsFromResource } from '@stoplight/prism-http';
+import { IPrismHttpServer } from '@stoplight/prism-http-server/src/types';
+import * as chokidar from 'chokidar';
+import { CreatePrismOptions } from './createServer';
+
+export function runPrism(createPrism: Function, options: CreatePrismOptions, spec: string) {
+  return createPrism(options).then((server: IPrismHttpServer) => {
+    if (server) {
+      const watcher = chokidar.watch(spec, { awaitWriteFinish: { stabilityThreshold: 500, pollInterval: 100 } });
```

Comment:
We also need to set the `ignoreInitial` to true, otherwise we get a reload event just when we loaded the file for the first time.

```suggestion
      const watcher = chokidar.watch(spec, { ignoreInitial:true, awaitWriteFinish: { stabilityThreshold: 500, pollInterval: 100 } });
```

Or what we could do (and it would be probably more correct) — we keep the flag to false and we remove the first "automatic" loading we're doing in yargs and we treat the first loading as a reload event as well. What do you think @lag-of-death ?

Reply:
@XVincentX, hm.. I think that https://github.com/stoplightio/prism/blob/38f1acfa8f2907a2d41b6d47ad84a48b02054025/packages/cli/src/util/runner.ts#L11 is the answer. `change` is the only event I listen to. I think that's all we need here. In case someone deletes the file, then we don't do a thing and keep the Prism running